### DISCUSSION TITLE
Flutter version?

### DISCUSSION THREAD

**Discussion by ishitb** [trakt/showly#561] [Ideas / Feature Requests]
Not really sure if you'd want to or not, but what do you think about making the application using flutter. A bit easier than Java and Kotlin. Can be used to release to iOS someday as well, and other platforms too. Since you have the app as open source, which is the greatest idea, I was thinking of starting the flutter version, but wanted your opinion since I'll be working on it only time to time and won't be able to give as much time, but still would love to try.
If you or anyone else wanna contribute, do let me know. It can turn into a really fun project.

**Comment by michaldrabik** [comment]
Privately I'm not a big fan of any cross platform solutions so I do not intend to do a Flutter one. 
I'm also pretty sure there is no way of achieving the same performance I got with current version.
I'd rather just work with some iOS friend of mine and release native IOS version one day.
